libfastboot/fastboot_transport.c

```diff
@@ -199,7 +231,11 @@ static EFI_STATUS fastboot_tcp_start(start_callback_t start_cb,
 	if (EFI_ERROR(ret))
 		return ret;
 
-	print_tcpip_information(&station_address);
+	ret = fastboot_get_host_ip(&station_address2);
```

Comment:
check whether it is running on a  KVM virtual by calling is_running_on_kvm().
for bare metal,  read host ip from smbios is not necessary